README.md

```diff
@@ -1,10 +1,12 @@
 BeVexed 1.1
 A maddeningly-addictive puzzle game.
 
+![BeVexed app](https://github.com/ayushmunot/BeVexed/blob/master/BeVexed.png?raw=true "BeVexed")
```

Comment:
You mustn't link to the screenshot at another site, as it can vanish any day. Do add it right beside the README.md file.
Also crop the screenshot pixel perfect to the window. Use Haiku's screenshot app so the background to the right of the yellow tab is transparent. Also take a shot from the game in action, i.e. some tiles on the left side.